CMakeLists.txt

```diff
@@ -1,6 +1,33 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.0.0)
 PROJECT(RedisAI)
```

Comment:
list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/opt/cmake/modules)
is gone? :-)

Reply:
Yes.